CONTRIBUTING.md

```diff
@@ -218,7 +218,7 @@ only pull requests targeted directly against master.
 ### Development Procedure
 
 - the latest state of development is on `master`
-- `master` must never fail `make test` or `make test_cli`
+- `master` must never fail `make test`
```

Comment:
```suggestion
- `master` must never fail `make lint test test-race`
```

This would ensure that race detection runs successfully too. Following line can be removed too.